components/tools/OmeroWeb/omeroweb/webclient/webclient_gateway.py

```diff
@@ -2177,18 +2177,12 @@ def groupSummary(self):
         @return:    {'leaders': list L{ExperimenterWrapper}, 'colleagues': list L{ExperimenterWrapper}}
         @rtype:     dict
         """
-        dropdown_menu = self._conn.getDropdownMenuSettings()
-        summary = self._conn.groupSummary(self.getId())
-        if "leaders" in dropdown_menu.keys():
-            self.leaders = summary["leaders"]
-            self.leaders.sort(key=lambda x: x.getLastName().lower())
-        if "colleagues" in dropdown_menu.keys():
-            self.colleagues = summary["colleagues"]
-            self.colleagues.sort(key=lambda x: x.getLastName().lower())
+        self.leaders, self.colleagues = self.groupSummary()
+        self.leaders.sort(key=lambda x: x.getLastName().lower())
+        self.colleagues.sort(key=lambda x: x.getLastName().lower())
```

Comment:
I removed sorting from gateway method as it should be really client responsibility